webapp/src/Controller/Jury/ContestController.php

```diff
@@ -907,4 +907,13 @@ private function checkTimezones(FormInterface $form): ?Response
         }
         return null;
     }
+
+    /**
+     * @Route("/deleteList", name="jury_contests_delete")
+     * @IsGranted("ROLE_ADMIN")
+     */
+    public function deleteListAction(Request $request): Response
+    {
+        return parent::deleteListActionHelper($request, Contest::class, 'jury_contests', 'contest');
```

Comment:
Why not:

```suggestion
        return $this->deleteListActionHelper($request, Contest::class, 'jury_contests', 'contest');
```

? I think that is more common?

Reply:
In an earlier version the function was overwritten, I agree this makes more sense with the current code.